src/main/java/org/andersen/model/Apartment.java

```diff
@@ -1,24 +1,36 @@
 package org.andersen.model;
 
+import jakarta.persistence.*;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.io.Serializable;
 
 @Data
```

Comment:
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/
please, read the article. don't use Data with Enitties))